specification/resources/resource-manager/Microsoft.Resources/stable/2022-12-01/subscriptions.json

```diff
@@ -934,6 +934,14 @@
     }
   },
   "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription.",
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
Revert it to be same as previous stable api-version 2021-01-01
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Resources/stable/2021-01-01/subscriptions.json#L821-L828

Reply:
@weidongxu-microsoft  This looks good. Please go ahead if you can get approval from Mike on Breaking Changes